assets/js/vendor/gridder.js

```diff
@@ -154,7 +154,7 @@
                     var nextItem = ($(".selectedItem").next().next());
 
                     htmlcontent += "<div class=\"gridder-navigation\">";
-                    htmlcontent += "<div class=\"nav-left\"><a href=\"#\" class=\"gridder-close\"><img src=\"/assets/img/ui/close-icon.png\"></a></div>";
+                    htmlcontent += "<div class=\"nav-left\"><a href=\"#\" class=\"gridder-close\">"+settings.closeText+"</a></div>";
```

Comment:
I think we need text for accessibility. Maybe change this to a button or give it a role of button and an `aria-label`?

Reply:
added `role` and `aria-label` in `onContent` as discussed.